### PR TITLE
Added zap scan to PR check to test it out

### PR DESCRIPTION
**Pull request by dbayly-freshworks**

### CODE REVIEW

**Comment by arranfw** [comment]
What's the procedure/plan to notify about these issues? Is the only way for now to navigate to the action and download the ZIP?

**Comment by dbayly-freshworks** [comment]
> What's the procedure/plan to notify about these issues? Is the only way for now to navigate to the action and download the ZIP?

That looks to be the plan. I could remove the -I flag (it was enabled on HCAP) to have the action fail, which would alert the dev running the action. What do you think?

File: .github/workflows/deploy-dev.yml [hunk 1]
```diff
@@ -65,3 +65,9 @@ jobs:
           make plan
           make deploy-infra
           make deploy-app
+
```
Comment (fwpushan) [inline .github/workflows/deploy-dev.yml:68]: Do we need to wait a bit to ensure deployment ?

Comment (arranfw) [reply .github/workflows/deploy-dev.yml:68]: I believe our TF process waits for cloudfront to be fully updated before it finishes

**Review by arranfw — APPROVED** [review]

**Pull request merged**